docs/voldoen-aan-wetten-en-regels/index.md

```diff
@@ -17,23 +17,8 @@ hide:
     <p>In augustus 2024 is de Verordening Artificiële Intelligentie in werking getreden. Lees meer over de AI-verordening of gebruik de beslishulp om te bepalen of de AI-verordening geldt voor jouw beoogde toepassing.</p>
     <ul>
       <li><a href="ai-verordening/">De AI-verordening in het kort</a></li>
-      <li><a href="https://ai-act-decisiontree.apps.digilab.network/">Gebruik de beslishulp AI-verordening</a></li>
     </ul>
-    <a href="https://ai-act-decisiontree.apps.digilab.network/" class="button button-primary">Voer beslishulp AI-verordening uit</a>
-    <!-- Modal Trigger -->
-    <p><a class="button" href="#popup1" id="openModalButton">Voer beslishulp AI-verordening uit</a></p>
-    <!-- Modal Structure -->
-    <div id="popup1" class="overlay">
-      <div class="popup">
-        <h2>Beslishulp AI-Verordening</h2>
-        <a class="close" href="javascript:void(0);" onclick="closeModal();">&times;</a>
-        <div class="content">
-          <div id="app">
-            <!-- Dynamisch script wordt hier geladen -->
-          </div>
-        </div>
-      </div>
-    </div>
+    <a href="https://ai-act-decisiontree.apps.digilab.network/" class="button md-button--secondary">Voer beslishulp AI-verordening uit</a>
```

Comment:
Willen we de beslisboom op een nieuwe tab openen?

Reply:
goede vraag!!! ik zat er ook over te denken dus laten we maar doen!